pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -157,12 +157,12 @@ rec {
 
   # CPAN mirrors.
   cpan = [
-    http://ftp.gwdg.de/pub/languages/perl/CPAN/
-    ftp://download.xs4all.nl/pub/mirror/CPAN/
+    https://ftp.gwdg.de/pub/languages/perl/CPAN/
+    https://cpan.metacpan.org/
+    https://cpan.perl.org/
     http://ftp.tuwien.ac.at/pub/CPAN/
     http://ftp.funet.fi/pub/CPAN/
```

Comment:
I think we can delete the `http` mirrors here, when we have better alternatives.

Reply:
They are gone for anoher reason: https://github.com/NixOS/nixpkgs/pull/52505
Do we need to left some ```http``` proxies for the case when https might not work (strict office firewall with DPI, etc) ?

Reply:
Not really. Our binary cache, most of the sources and a good part of the internet these days are barely usable without https.